src/components/ForecastMainBanner.svelte

```diff
@@ -0,0 +1,112 @@
+      <div class="flex justify-center items-center p-4 md: pt-5">
+        <Spinner />
+      </div>
+    {:else if weatherData.hasError}
+      <div class="flex justify-center items-center p-4 md: pt-5">
+        <p>No data received</p>
+      </div>
+    {:else}
+      <div class="flex justify-between items-center">
+        {#if weatherData.weatherCode !== null && weatherCodes[weatherData.weatherCode]?.[dayPeriod]?.image}
+          <div class="basis-1/2 md:basis-1/4">
+            <span>
+              <img
+                class="min-h-24 h-24 md:h-32"
+                src={weatherCodes[weatherData.weatherCode][dayPeriod].image}
+                alt="weather"
+              />
+            </span>
+          </div>
+        {/if}
+
+        <div
+          class="forcast-info flex flex-col md:flex-row justify-between items-left basis-1/2 md:basis-3/4"
+        >
+          {#if weatherData.temperature !== null}
+            <div class="flex mt-2 items-end">
+              <span class="text-4xl md:text-5xl">
+                <img class="h-8 md:h-10" src={thermostat} alt="thermostat" />
+              </span>
+              <span class="text-red-600 text-4xl md:text-5xl">
+                {weatherData.temperature}°
+              </span>
+              {#if weatherData.feelsLike !== null}
+                <span class="ml-2 md:ml-4 text-sm md:text-lg">
+                  Feels like
+                  <span class="text-red-600">{weatherData.feelsLike}°</span>
+                </span>
+              {/if}
+            </div>
+          {/if}
+
+          {#if weatherData.precipitation !== null}
+            <div class="flex mt-2 items-end">
+              <span>
+                <img class="h-8 md:h-10" src={umbrella} alt="umbrella" />
+              </span>
+              <span class="ml-2 md:ml-4 text-4xl md:text-5xl text-blue-500"
+                >{weatherData.precipitation}</span
+              >
+              <span>&nbsp;mm</span>
+            </div>
+          {/if}
+
+          {#if weatherData.windSpeed !== null}
+            <div class="flex mt-2 items-end">
+              <span>
+                <img class="h-8 md:h-10" src={air} alt="air" />
+              </span>
+              <span class="ml-2 md:ml-4 text-4xl md:text-5xl">
+                {weatherData.windSpeed}
+              </span>
+              {#if weatherData.windGusts !== null}
+                <span class="ml-2 text-sm md:text-lg">
+                  ({weatherData.windGusts}) m/s
+                </span>
+              {/if}
+              <span>
+                <img
+                  class="h-8 md:h-10 relative z-[-10]"
+                  src={arrow}
+                  alt="arrow"
+                  style="transform: rotate({weatherData.windDirection}deg); transition: transform 0.5s;"
+                />
+              </span>
+            </div>
+          {/if}
+        </div>
+      </div>
+    {/if}
+  </div>
+</div>
+
+<style>
+  .forecast-panel {
```

Comment:
can something like: https://tailwindcss.com/docs/box-shadow
be used here?

Reply:
I've tried to use this property, but unfortunately it applies only 3 sides with shadow by default. However I can do it in this way:
`shadow-[0px_5px_15px_rgba(0,0,0,0.35)]`
Is it better?

Reply:
Yes, looks good